diff --git a/node_modules/unplugin-vue/dist/chunk-XENELXXE.mjs b/node_modules/unplugin-vue/dist/chunk-XENELXXE.mjs
index 97516fc..3f7dbd8 100644
--- a/node_modules/unplugin-vue/dist/chunk-XENELXXE.mjs
+++ b/node_modules/unplugin-vue/dist/chunk-XENELXXE.mjs
@@ -669,7 +669,7 @@ async function transformStyle(code, descriptor, index, options, context, filenam
     });
     return null;
   }
-  const map = result.map ? formatPostcssSourceMap(result.map, filename) : { mappings: "" };
+  const map = result.map ? formatPostcssSourceMap(result.map, filename) : false;
   return {
     code: result.code,
     map
