diff --git a/node_modules/unplugin/dist/index.js b/node_modules/unplugin/dist/index.js
index 89d0cc6..2e09978 100644
--- a/node_modules/unplugin/dist/index.js
+++ b/node_modules/unplugin/dist/index.js
@@ -1145,7 +1145,7 @@ function getEsbuildPlugin(factory) {
                 map.sourcesContent = [code];
               map = fixSourceMap(map);
               code += `
-//# sourceMappingURL=${map.toUrl()}`;
+/* sourceMappingURL=${map.toUrl()} */`;
             }
             return { contents: code, errors, warnings, loader: guessLoader(args.path), resolveDir };
           }
@@ -1174,7 +1174,7 @@ function getEsbuildPlugin(factory) {
                 map.sourcesContent = [code];
               map = fixSourceMap(map);
               code += `
-//# sourceMappingURL=${map.toUrl()}`;
+/* sourceMappingURL=${map.toUrl()} */`;
             }
             return { contents: code, errors, warnings, loader: guessLoader(args.path), resolveDir };
           }
diff --git a/node_modules/unplugin/dist/index.mjs b/node_modules/unplugin/dist/index.mjs
index bbac5da..79778e4 100644
--- a/node_modules/unplugin/dist/index.mjs
+++ b/node_modules/unplugin/dist/index.mjs
@@ -1113,7 +1113,7 @@ function getEsbuildPlugin(factory) {
                 map.sourcesContent = [code];
               map = fixSourceMap(map);
               code += `
-//# sourceMappingURL=${map.toUrl()}`;
+/* sourceMappingURL=${map.toUrl()} */`;
             }
             return { contents: code, errors, warnings, loader: guessLoader(args.path), resolveDir };
           }
@@ -1142,7 +1142,7 @@ function getEsbuildPlugin(factory) {
                 map.sourcesContent = [code];
               map = fixSourceMap(map);
               code += `
-//# sourceMappingURL=${map.toUrl()}`;
+/* sourceMappingURL=${map.toUrl()} */`;
             }
             return { contents: code, errors, warnings, loader: guessLoader(args.path), resolveDir };
           }
